src/Pages/Driver/enterRoute.tsx

```diff
@@ -19,9 +18,9 @@ const MARKS = {
   0: '0 km',
```

Comment:
I wonder... Does it make sense to even have a "0 km" detour option? I don't think this sends the correct message to drivers; they'll always have to make a bit of a detour, except for maybe 0.5% of the cases.
What do you think, @conradbielitz , from a business perspective?

Reply:
Makes sense for me too!